src/sql/workbench/contrib/notebook/browser/cellViews/outputArea.component.ts

```diff
@@ -38,7 +38,6 @@ export class OutputAreaComponent extends AngularDisposable implements OnInit {
 			this._register(this.cellModel.onOutputsChanged(e => {
 				if (!(this._changeRef['destroyed'])) {
 					this._changeRef.detectChanges();
-					this._changeRef.detach();
```

Comment:
If there aren’t any changes to the components, this._changeRef.detectChanges() won’t re-render / re-layout anything. 

Previously, the gridOutput.component was being re-rendered because one of the @inputs was being changed. Since the other outputs are not dynamic, they shouldn’t be re-rendered.